build/ci/cspell.json

```diff
@@ -0,0 +1,223 @@
+﻿{
+	"version": "0.2",
+	"language": "en",
+	"words": [
+		"Avalonia",
```

Comment:
We are specifying `Avalonia` somewhere?

Reply:
@carldebilly I based this file on the one we started on Uno for the first repo with spell-checking.
@jeromelaban it it worth adjusting the list based on the current repo only?

Reply:
It's a common list, so that's fine.